tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -34,6 +35,37 @@ public class WasmAppBuilder : Task
     SortedDictionary<string, Assembly>? _assemblies;
     Resolver? _resolver;
 
+    private class WasmAppConfig
+    {
+        public string assembly_root { get; set; } = "managed";
```

Comment:
Instead of using the json naming in C# maybe we should use proper C# names and use `JsonPropertyName`: https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-how-to#customize-individual-property-names

Reply:
they are private classes meant only for serialization.

Reply:
I know but it still looks a bit weird in C# :)

Reply:
looks weird in js too ;)

Reply:
Perhaps we should look at the structure and naming here.